### PR TITLE
[WIP] Added django_rq and ``ext`` queue for extensions

### PR DESCRIPTION
**Pull request by ar4s**
It's first step to microservices. It's closely related with #1791.

### CODE REVIEW

File: vagrant/upstart/rqworker.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+start on runlevel [2345]
+stop on runlevel [!2345]
+
+respawn
+chdir /home/vagrant
+exec bin/ralph rqworker ext--settings=ralph.settings.dev
```
Comment (mkurek) [inline vagrant/upstart/rqworker.conf]: don't you miss space here?

Comment (ar4s) [inline vagrant/upstart/rqworker.conf]: yep :)

File: src/ralph/settings/base.py [hunk 2]
```diff
@@ -168,3 +169,12 @@
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',  # noqa
     'PAGE_SIZE': 10,
 }
+
+RQ_QUEUES = {
+    'ext': {
+        'HOST': 'localhost',
```
Comment (mkurek) [inline src/ralph/settings/base.py]: use `os.environ.get('REDIS_HOST', 'localhost')` and make it working in docker too (in docker compose run redis container)

File: src/ralph/settings/base.py [hunk 3]
```diff
@@ -168,3 +169,12 @@
     'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',  # noqa
     'PAGE_SIZE': 10,
 }
+
+RQ_QUEUES = {
+    'ext': {
+        'HOST': os.environ.get('REDIS_HOST', 'localhost'),
+        'PORT': os.environ.get('REDIS_PORT', '6379'),
+        'DB': 0,
+        'ASYNC': False,
```
Comment (mkurek) [inline src/ralph/settings/base.py:178]: why not async?

Comment (ar4s) [inline src/ralph/settings/base.py:178]: because we want run task and gets results immediately

**Pull request closed without merging**